### ISSUE TITLE
[a11y] 9.1.4.3 Text contrasts are sufficient

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10726]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-4-3-kontraste-von-texten-ausreichend

Note: For texts (also in the focused state) under 18 points (or for bold texts under
bold text below 14 point), the contrast ratio between foreground and background color should be
background color should be at least 4.5:1. For large fonts, the contrast ratio should be
should be 3:1 or greater.

### Personal

- [ ] In the "Actions" area, the shortcuts currently do not fulfill the contrast
contrast requirements for texts. The current contrast value is 3.93:1.
![image](https://github.com/owncloud/web/assets/132444655/d424e813-1c20-4412-a6b0-fc5cc5980261)

- [ ] The error messages do not meet the contrast requirements for texts. The
current contrast value is 3.99:1
![image](https://github.com/owncloud/web/assets/132444655/40f02ea1-f304-42bc-94f7-0cc254c5b9d2)

### Shares

- [ ] The labels on the "Accept" buttons do not meet the contrast requirements for
contrast requirements for text, as the contrast ratio of 4.04:1 is below the minimum value of
minimum value of 4.5:1. 
![image](https://github.com/owncloud/web/assets/132444655/d1539eab-77e8-4be9-a1d8-7029569145af)

### Deleted Files

- [ ] Deactivated elements in the table do not meet the contrast requirements for texts.
The current contrast value is 1.67:1.
![image](https://github.com/owncloud/web/assets/132444655/34da925c-ee16-495c-9384-38ff56af5cc0)

**Comment by pascalwengerter** [comment]
@tbsbdr needs design rework (light&dark mode)
